Eatery Blue/UI/ListScreen/ListViewController.swift

```diff
@@ -206,7 +207,7 @@ class ListViewController: UIViewController {
         tableView.contentOffset = CGPoint(x: 0, y: -tableView.contentInset.top)
     }
 
-    func updateEateries(_ eateries: [Eatery]) {
+    func updateEateries(_ eateries: [Eatery], allEateries: [Eatery]) {
```

Comment:
`allEateries` is not used anywhere here. I don't think you need this at all.

Reply:
good catch. its supposed to update self.allEateries and would have produced a pretty bad bug in which the compare menus button brings up nothing (because its defaulted to empty array.